docs/watcher/run-watcher.md

```diff
@@ -0,0 +1,611 @@
+
+## Costs
+
+The costs of running a Watcher include the following components:
+- A full Ethereum node (local or ETH provider)
+- VPS, bare-metal server, or local machine that matches [the minimum hardware requirements](#minimum-hardware-requirements)
+- DevOps setup and maintenance fee
+
+## Setup
+
+Before installing Watcher service on a server or your local machine, you should have a few things setup:
+
+### 1. Log in to Your Server
+
+If you're planning to install a Watcher on a remote server (VPS or bare-metal), make sure to have an active session open before proceeding with the guide.
+
+The configuration process takes a significant amount of time and may require help from your DevOps team. This step is fully covered in the [Manage VPS](/watcher/manage-vps) guide.
+
+You can log in using the following command from your terminal or command prompt:
+
+```
+ssh $USER@$REMOTE_SERVER -p $PORT
+```
+
+> - `$USER` - the name of the user with root privileges used to log into the remote server. Default: root.
+> - `$REMOTE_SERVER` - an IP address of your remote server.
+> - `$PORT` - a port used to connect to the server. Default: 22.
+
+### 2. Check TCP ports
+
+Make sure you have don't have any of the services running on one of the following ports: 7434, 7534, 5432. The ports are used as follows:
+- 7434: `elixir-omg_watcher_1`, a light-weight Watcher to ensure the security of funds deposited into the childchain
+- 7534: `elixir-omg_watcher_info_1`, a convenient and performant API to the child chain data
+- 5432: `elixir-omg_postgres_1`, a PostgreSQL database that stores transactions and contains data needed for challenges and exits
+
+You can use `lsof`, `netstat` or other alternatives to check occupied ports:
+
+```
+sudo lsof -i -n -P | grep LISTEN
+```
+
+If you found one of the ports is already in use, kill the process the port is occupied with as follows:
+
+```
+sudo lsof -t -i:$PORT
+sudo kill -9 $PID
+```
+
+> - `$PORT` - a port to clear from other processes
+> - `$PID` - process ID listening on a defined port
+
+If you're already using Docker, verify these ports are free too:
+
+```
+docker ps
+```
+
+### 3. Update Ubuntu Packages
+
+If you're using Ubuntu-based system, make sure to update your packages:
+
+```
+sudo apt-get update
+```
+
+## Install
+
+Running a Watcher locally is recommended for testing purposes only. For production, you should use VPS or bare-metal server. This allows to increase uptime, reduce latency, and configure advanced security measures for your instance.
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!-- Bare-metal Release -->
+
+### 1. Install Dependencies
+
+#### 1.1 Install Erlang and Elixir
+
+The current guide shows how to install Erland and Elixir using [`asdf`](https://asdf-vm.com) to allow managing multiple runtime versions. Asdf relies on several libraries that you may need to install first:
+
+```
+sudo apt-get install libssl-dev make automake autoconf libncurses5-dev gcc unzip
```

Comment:
in our process, we normally simply use "bin/setup" script from elixir-omg

Reply:
I was using those mostly because they were missing during the installation for whatever reason.

Reply:
I can remove them, maybe it was only the problem with my server. Not sure